src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,42 @@
-package core.basesyntax;
+package core.colors;
+
+import java.util.Random;
 
 public class ColorSupplier {
+    enum Colors {
```

Comment:
1. move enum to a separate file
2. enum name should be singular (Color)
3. enum values should be written as constants (MEDIUM_PURPLE)

Reply:
refactored all, suitably to remarks